internal/stores/contracts.go

```diff
@@ -3,13 +3,28 @@ package stores
 import (
 	"encoding/json"
 	"errors"
+	"fmt"
+	"math/big"
 	"os"
 	"path/filepath"
 	"sync"
 
+	"go.sia.tech/siad/crypto"
+
 	"go.sia.tech/renterd/internal/consensus"
 	rhpv2 "go.sia.tech/renterd/rhp/v2"
 	"go.sia.tech/siad/types"
+	"gorm.io/gorm"
+)
+
+var (
+	// ErrContractNotFound is returned when a contract can't be retrieved from the
+	// database.
+	ErrContractNotFound = errors.New("couldn't find contract with specified id")
```

Comment:
`nit` no need to add "with specified id", should be added as context to this error